src/components/function-summary/result.tsx

```diff
@@ -8,13 +8,25 @@ import { cvToJSON, hexToCV } from '@stacks/transactions';
 
 interface FunctionSummaryResultProps {
   result: Transaction['tx_result'];
-  txStatus: string | undefined;
+}
+interface ReprValueProps {
+  type: string;
+  value: string | number | (string | number)[];
 }
 
-export const FunctionSummaryResult = ({ result, txStatus }: FunctionSummaryResultProps) => {
+export const FunctionSummaryResult = ({ result }: FunctionSummaryResultProps) => {
   if (!result) return null;
   const { success, type, value } = cvToJSON(hexToCV(result.hex));
   const hasType = !type?.includes('UnknownType');
+
+  const getReprValue = ({ type, value }: ReprValueProps) => {
+    let reprValue = value ?? 'none';
```

Comment:
I would add `.toString();` to the value here to match the old code. This part is pretty sensitive to changes.